src/components/CloudHeader/CloudHeader-test.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import CloudHeader from './CloudHeader';
+import CloudHeaderWrapper from './CloudHeaderWrapper';
+import CloudHeaderMenu from './CloudHeaderMenu';
+import CloudHeaderLogo from './CloudHeaderLogo';
+import CloudHeaderList from './CloudHeaderList';
+import CloudHeaderListItem from './CloudHeaderListItem';
+import { shallow, mount } from 'enzyme';
+
+describe('CloudHeader', () => {
+  describe('Renders as expected', () => {
+    const header = shallow(<CloudHeader className="some-class" />);
+    it('has the expected classes', () => {
+      expect(header.hasClass('bx--cloud-header')).toEqual(true);
+    });
+    it('should add extra classes that are passed via className', () => {
+      expect(header.hasClass('some-class')).toEqual(true);
+    });
+  });
+});
+
+describe('CloudHeaderWrapper', () => {
```

Comment:
This might sound tedious, but could each component have it's own test file?

Reply:
Done